src/lib/core/redux/slices/rtcConnection/index.ts

```diff
@@ -283,6 +283,37 @@ export const selectRtcStatus = (state: RootState) => state.rtcConnection.status;
 /**
  * Reactors
  */
+startAppListening({
+    predicate: (_action) => {
+        return (
+            Object.entries(rtcAnalyticsCustomEvents)
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                .map(([_, value]) => value.actionType)
+                .includes(_action.type)
+        );
```

Comment:
Redux toolkit has this helper: `isAnyOf`, which you can use to check if an action is of a specific type.
I think we can do something like this:
```js
  const actions = Object.values(rtcAnalyticsCustomEvents).map((e) => e.action);
  const isRtcEvent = isAnyOf(...actions);

  return isRtcEvent(action);
```
Requires some change in the events obj, but I think it should be straightforward. Just import the action creators directly and use those as the `action` instead of the string.